src/component/header/Header.jsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from 'react';
+import headerJSON from '../../data/header.json';
+
+const Header = () => {
+    const [isChanged, setIsChanged] = useState(false);
+    const [isMenuOpen, setMenuOpen] = useState(false);
+
+    const toggleMenu = () => {
+        setMenuOpen(!isMenuOpen);
+        setIsChanged(!isChanged);
+    };
+
+    useEffect(() => {
+        const handleResize = () => {
+            if (window.innerWidth >= 768) {
+                setMenuOpen(false);
+                setIsChanged(false);
+            }
+        };
+
+        window.addEventListener('resize', handleResize);
+
+        return () => {
+            window.removeEventListener('resize', handleResize);
+        };
+    }, []);
+
+    return (
+        <div className="w-full fixed">
+            <div className="flex flex-row justify-between p-6 md:p-0  md:py-5 z-50 md:px-16 lg:px-20 xl:px-28 2xl:px-36 items-center  bg-white bg-opacity-50 backdrop-blur-md">
```

Comment:
Please check the formatting seems like there is some extra space `md:p-0 md:py-5` and `items-center bg-white`